src/game.cpp

```diff
@@ -10838,24 +10838,40 @@ void game::wield( int pos )
         add_msg( m_info, _( "You cannot unwield your %s." ), u.weapon.tname().c_str() );
         return;
     }
-    if( pos == INT_MIN ) {
-        pos = inv_for_all( _( "Wield item" ), _( "You have nothing to wield." ) );
+
+    item_location loc;
+    if( pos != INT_MIN ) {
+        loc = std::move( item_location( u, &u.i_at( pos ) ) );
```

Comment:
Nope, it was originally copy able but required to be ephemeral https://github.com/CleverRaven/Cataclysm-DDA/commit/5d558f87732e907db4fb4bf02aca4317d1e49b5c

It became un-copyable here https://github.com/CleverRaven/Cataclysm-DDA/commit/f0609394a8b3fdd39c19ccf41661c1aafc3cfb18 to support serialization.

This isn't relevant for this PR anyway, you're just using it as written and I forgot about it being broken like this.